app/install.php

```diff
@@ -662,22 +662,24 @@ function printStep5() {
 	</head>
 	<body>
 
-<div class="header">
+<header class="header">
 	<div class="item title">
 		<h1><a href="index.php"><?= _t('install.title') ?></a></h1>
 		<h2><?= _t('install.step', STEP) ?></h2>
 	</div>
-</div>
-
-<div id="global">
-	<ul class="nav nav-list aside">
-		<li class="nav-header"><?= _t('install.steps') ?></li>
-		<li class="item<?= STEP == 0 ? ' active' : '' ?>"><a href="?step=0"><?= _t('install.language') ?></a></li>
-		<li class="item<?= STEP == 1 ? ' active' : '' ?>"><a href="?step=1"><?= _t('install.check') ?></a></li>
-		<li class="item<?= STEP == 2 ? ' active' : '' ?>"><a href="?step=2"><?= _t('install.bdd.conf') ?></a></li>
-		<li class="item<?= STEP == 3 ? ' active' : '' ?>"><a href="?step=3"><?= _t('install.conf') ?></a></li>
-		<li class="item<?= STEP == 4 ? ' active' : '' ?>"><a href="?step=4"><?= _t('install.this_is_the_end') ?></a></li>
-	</ul>
+</header>
+
+<main id="global">
+	<nav class="nav nav-list aside">
+		<ul>
+			<li class="nav-header"><?= _t('install.steps') ?></li>
+			<li class="item<?= STEP == 0 ? ' active' : '' ?>"><a href="?step=0"><?= _t('install.language') ?></a></li>
+			<li class="item<?= STEP == 1 ? ' active' : '' ?>"><a href="?step=1"><?= _t('install.check') ?></a></li>
+			<li class="item<?= STEP == 2 ? ' active' : '' ?>"><a href="?step=2"><?= _t('install.bdd.conf') ?></a></li>
+			<li class="item<?= STEP == 3 ? ' active' : '' ?>"><a href="?step=3"><?= _t('install.conf') ?></a></li>
+			<li class="item<?= STEP == 4 ? ' active' : '' ?>"><a href="?step=4"><?= _t('install.this_is_the_end') ?></a></li>
+		</ul>
+	</nav>
 
 	<div class="post">
```

Comment:
Is #global the main or this one?

Reply:
both would be fine.

#global would be better in sync to https://github.com/FreshRSS/FreshRSS/blob/edge/app/views/index/global.phtml (where the class `global` is changed to `<main>`)

Reply:
Alright, let's stick to #global then.